### ISSUE TITLE
sample: 

### ISSUE THREAD

**Issue by gilligan** [nix-community/nixpkgs-fmt#162]
## Input
```nix
{
      inherit (import ./pep425.nix {
        inherit lib python;
        inherit (pkgs) stdenv;
      }) selectWheel;
}
```

## Output
```nix
{
  inherit (import ./pep425.nix {
    inherit lib python;
    inherit (pkgs) stdenv;
  }) selectWheel
    ;
}

```

## Desired output

```nix
{
  inherit (import ./pep425.nix {
    inherit lib python;
    inherit (pkgs) stdenv;
  }) selectWheel;
}
```

The `;` should not be on a separate line.

**Comment by zimbatm** [comment]
indeed. nixpkgs-fmt usually leaves the position of `;` up to the user, it shouldn't enforce the line break here.

**Comment by Rizary** [comment]
fixed by: #172
